### DISCUSSION TITLE
Is there an official QQ group or WeChat group for competition?

### DISCUSSION THREAD

**Discussion by JuneWaySue** [PaddlePaddle/PaddleSpatial#66] [BaiduKDDCup2022] [answered]
hi organizer~
Is there an official QQ group or WeChat group for competition?
I think that it will make communication much easier.
Thanks!!

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

Considering KDD Cup as an international contest, for the fairness of participants from all countries over the world, we will not open any official QQ group or WeChag group. Any communication can be done by the GitHub Discussions or emails. 

Thanks for your understanding. 

Regards
